src/main/java/org/apache/sling/event/impl/jobs/stats/StatisticsManager.java

```diff
@@ -57,9 +64,15 @@ public synchronized void reset() {
 
     };
 
+    /** Gauges for the global job statistics. */
+    private GaugeSupport globalGauges;
+
     /** Statistics per topic. */
     private final ConcurrentMap<String, TopicStatistics> topicStatistics = new ConcurrentHashMap<>();
 
+    /** Gauges for the statistics per topic. */
+    private final ConcurrentMap<String, GaugeSupport> topicGauges = new ConcurrentHashMap<>();
```

Comment:
I understand we do the other maps as `ConcurrentMap` in this class too and that those other maps do have potential for race-conditions. Then again, for those other cases it is not "so bad" as we're only talking about statistics. 
With `GaugeSupport` it looks like `metricRegistry.register()` could throw an `IllegalArgumentException` and that would be problematic